hexkit/providers/mongodb/testutils.py

```diff
@@ -33,9 +34,37 @@
 class MongoDbFixture:
     """Yielded by the `mongodb_fixture` function"""
 
+    client: MongoClient
     config: MongoDbConfig
     dao_factory: MongoDbDaoFactory
 
+    def empty_collections(
+        self,
+        exclude_collections: Optional[Union[str, list[str]]] = None,
+    ):
+        """Drop all mongodb collections in the database.
+
+        You can also specify collection(s) that should be excluded
+        from the operation, i.e. collections that should be kept.
+        """
+        db_name = self.config.db_name
+        if exclude_collections is None:
+            exclude_collections = []
+        if isinstance(exclude_collections, str):
+            exclude_collections = [exclude_collections]
+        excluded_collections = set(exclude_collections)
+        try:
+            collection_names = self.client[db_name].list_collection_names()
+            for collection_name in collection_names:
+                if collection_name not in excluded_collections:
+                    self.client[db_name].drop_collection(collection_name)
+        except (ExecutionTimeout, OperationFailure) as error:
+            raise RuntimeError(
+                f"Could not drop collection(s) of Mongo database {db_name}"
+            ) from error
+        for collection_name in self.client[db_name].list_collection_names():
+            self.client[db_name].drop_collection(collection_name)
```

Comment:
This should be removed, right?
Would drop all collections either way and has no error handling

Reply:
Yeah I missed that